packages/Search/src/details/DTK_DetailsAlgorithms.hpp

```diff
@@ -192,6 +195,18 @@ Point return_centroid( Box const &box )
 KOKKOS_INLINE_FUNCTION
 Point return_centroid( Sphere const &sphere ) { return sphere.centroid(); }
 
+// transformation that maps the unit cube into a new axis-aligned box
+KOKKOS_INLINE_FUNCTION
+void translateAndScale( Point const &in, Point &out, Box const &ref )
+{
+    for ( int d = 0; d < 3; ++d )
```

Comment:
Do you not want to add a check that `Box` is not degenerate?

Reply:
No seatbelt

Reply:
But can you guarantee during the runtime that it never happens? What if the cloud of point is on axis-aligned plane?